config/config_test.go

```diff
@@ -6,6 +6,21 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestValidWebhookURL(t *testing.T) {
+	t.Run("empty allowList", func(t *testing.T) {
+		var cfg Config
+
+		assert.True(t, cfg.ValidWebhookURL("http://api.example.com"))
+	})
+	t.Run("checks if webhook urls are allowed", func(t *testing.T) {
+		var cfg Config
+		cfg.Webhook.AllowList = append(cfg.Webhook.AllowList, "http://api.example.com")
+
+		assert.True(t, cfg.ValidWebhookURL("http://api.example.com:5555/path"))
+		assert.False(t, cfg.ValidWebhookURL("http://example.com"))
+	})
```

Comment:
These can just be run flat
```suggestion
	var cfg Config

	assert.True(t, cfg.ValidWebhookURL("http://api.example.com"))

	cfg.Webhook.AllowList = append(cfg.Webhook.AllowList, "http://api.example.com")

	assert.True(t, cfg.ValidWebhookURL("http://api.example.com:5555/path"))
	assert.False(t, cfg.ValidWebhookURL("http://example.com"))
```